storage/campaignfailedlog.go

```diff
@@ -0,0 +1,37 @@
+package storage
+
+import (
+	"fmt"
+
+	"github.com/mailbadger/app/entities"
+)
+
+// CreateReport creates a report.
+func (db *store) CreateCampaignFailedLog(l *entities.CampaignFailedLog) error {
+	return db.Create(l).Error
+}
+
+func (db *store) LogFailedCampaign(c *entities.Campaign, log *entities.CampaignFailedLog) error {
```

Comment:
Can we maybe create the log entity inside the method? It would be cleaner that way.